tests/scenario_invoice.rst

```diff
@@ -228,3 +229,31 @@ Create credit_note IN on custom journal::
     >>> invoice.click('post')
     >>> invoice.number
     'C2'
+
+Set the sequence number::
+
+    >>> sequence = fiscalyear.post_move_sequence
+    >>> sequence.number_next = 10
+    >>> sequence.save()
+
+Renew fiscalyear using the wizard::
+
+    >>> Fiscal_year = Model.get('account.fiscalyear')
+    >>> fiscal_years = len(Fiscal_year.find([]))
+    >>> fiscal_years
+    1
+    >>> renew_fiscalyear = Wizard('account.fiscalyear.renew')
+    >>> renew_fiscalyear.form.reset_sequences = False
+    >>> renew_fiscalyear.execute('create_')
+    >>> new_fiscalyear, = renew_fiscalyear.actions[0]
+    >>> len(new_fiscalyear.periods)
+    12
+    >>> int(new_fiscalyear.post_move_sequence.number_next)
+    10
+    >>> fiscal_years = len(Fiscal_year.find([]))
+    >>> fiscal_years
+    2
+    >>> new_fiscal_year = Fiscal_year.find(["id", "=", 2])[0]
+    >>> new_sequences = new_fiscal_year.journal_sequences[0].id, new_fiscal_year.journal_sequences[1].id
+    >>> str(new_sequences)
+    '(3, 4)'
```

Comment:
Check that they exist and that they're different from the ones in the previous year.